tools/local-snapshot-runner-app/src/fakeUrlResolver.ts

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IRequest } from "@fluidframework/core-interfaces";
+import { IContainerPackageInfo, IResolvedUrl, IUrlResolver, } from "@fluidframework/driver-definitions";
+import { IOdspResolvedUrl } from "@fluidframework/odsp-driver-definitions";
+
+/**
+ * Fake URL resolver that returns hard coded values on every request
+ */
+export class FakeUrlResolver implements IUrlResolver {
+
+    public async resolve(request: IRequest): Promise<IResolvedUrl | undefined> {
+        const fakeOdspResolvedUrl: IOdspResolvedUrl = {
+            type: "fluid",
+            odspResolvedUrl: true,
+            id: "1",
```

Comment:
I'd use some unique strings instead of "1". Some of these will get to telemetry stream and someone will wonder where they are coming from. It's so much easier to find unique strings...